<?xml version="1.0"?>
<launch>
	<rosparam file="$(find kari_localization)/yaml/maps.yaml" />
	<node pkg="kari_localization" type="map_alignment_ver_tsukuba" name="alignment" output="screen" />
        <param name="laser_voxel_size" value="0.3" />
        <param name="map_voxel_size" value="0.3" />
        <param name="map_limit" value="20.0" />
        <param name="laser_limit" value="20.0" />
        <param name="init_x" value="0.0" />
        <param name="init_y" value="0.0" />
        <param name="init_z" value="0.0" />
        <param name="init_yaw" value="0.0" />


		<param name="odom_topic" value="/lcl_ekf" />
		<param name="ndt_topic" value="/sq_ndt_data" />

		<param name="lidar_topic" value="/cloud/lcl" />


	<node pkg="kari_localization" type="get_height" name="get_height" />
</launch>


